### PR TITLE
added assets from robotec-agriculture-gem

### PR DESCRIPTION
**Pull request by Jakub-Krakowiak**

### CODE REVIEW

File: RobotecSmallContainersAssets/NOTICE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-The assets in this repository are based on the assets prepared for ROSCon 2023 Demo located under https://github.com/RobotecAI/ROSCon2023Demo (accessed on July 24, 2024).
+The assets in this repository are based on the assets prepared for ROSCon 2023 Demo located under https://github.com/RobotecAI/ROSCon2023Demo (accessed on July 24, 2024) and [robotec-agriculture-gem](https://github.com/RobotecAI/robotec-agriculture-gem/).
 The base assets were licensed under Apache 2.0 license and the same license remains for this work.
```
Comment (pawelbudziszewski) [inline RobotecSmallContainersAssets/NOTICE:3]:
```suggestion

Both "Football" models were obtained from PolyHaven (https://polyhaven.com/a/football) and have CC0 license.

The "Barrel" model was obtained from PolyHaven (https://polyhaven.com/a/Barrel_02) and it has CC0 license.

The "CardboardBox" model was obtained from PolyHaven (https://polyhaven.com/a/cardboard_box_01) and it has CC0 license.

```

**Comment by pawelbudziszewski** [comment]
2 more things:

1. Rename `FootballInflated` to `Football`
2. Move the origin to the center of the ball. It will be easier to rotate it on the scene.

File: RobotecSmallContainersAssets/Assets/Barrel/Models/Barrel.fbx
Comment (jhanca-robotecai) [inline RobotecSmallContainersAssets/Assets/Barrel/Models/Barrel.fbx:1]: It looks like we have two barrels: a barrel and a plastic barrel.
image

**Review by jhanca-robotecai — CHANGES_REQUESTED** [review]
- notice information should be reworked
- barrels should be cleaned
- football origins should be fixed

**Comment by jhanca-robotecai** [comment]
> 2. Move the origin to the center of the ball. It will be easier to rotate it on the scene.

I disagree - this is against our agreement that we put the origin at the point that connects to the scene. I.e. the center of the bottom plane for objects that can be put on the ground or the center of the sidewall for objects that should be hung on a wall.

Ball with an origin in the center of it changes the position after spawning, as the collider of the ball interacts with the collider of the floor. This should not happen.

File: RobotecSmallContainersAssets/Assets/Barrel/Models/Barrel.fbx
Comment (pawelbudziszewski) [reply RobotecSmallContainersAssets/Assets/Barrel/Models/Barrel.fbx:1]: Two barrels mean more diversity. I'd leave them both. Named `PlasticBarrel1` and `PlasticBarrel2`

Comment (Jakub-Krakowiak) [reply RobotecSmallContainersAssets/Assets/Barrel/Models/Barrel.fbx:1]: Maybe name the new one PlasticBarrelWithHandles

**Comment by pawelbudziszewski** [comment]
> > 2. Move the origin to the center of the ball. It will be easier to rotate it on the scene.
> 
> I disagree - this is against our agreement that we put the origin at the point that connects to the scene. I.e. the center of the bottom plane for objects that can be put on the ground or the center of the sidewall for objects that should be hung on a wall.
> 
> Ball with an origin in the center of it changes the position after spawning, as the collider of the ball interacts with the collider of the floor. This should not happen.

The ball can rotate. In such cases, we can place the origin in the rotation center. Or at least this is my opinion.
The other thing is, that rotating the ball may give more visual diversity - it's much more convenient if the origin is in the center.
However if the origin in the center causes any issues when spawning, it should be moved to the bottom.

**Review by jhanca-robotecai — APPROVED** [review]
I fixed the problems I found during the code review, I do not have any more comments.

**Comment by jhanca-robotecai** [comment]
> The ball can rotate. In such case, we can place the origin in the rotation center. Or at least this is my opinion.
> The other thing is, that rotating the ball may give more visual diversity - it's much more convenient if the origin is in the center.
> However if the origin in the center causes any issues when spawning, it should be moved to the bottom.

I moved it back to the bottom. I see your point in making it more diverse by rotating, but as long as we do not simulate a pool full of balls, we do not have to rotate them at all.

**Review by pawelbudziszewski — APPROVED** [review]
There was an inconsistency in barrel prefabs. No both are dynamic rigid bodies and have consistent names.
I also access accessibility dates to model URLs (in NOTOCEs)

**Pull request merged**